### PR TITLE
由 开发框架-安卓开发框架 合并迁移至 手机和移动应用开发-安卓组件和项目

### PR DESCRIPTION
**Pull request by QYFabc**

### CODE REVIEW

File: collections.yml [hunk 1]
```diff
@@ -57,7 +57,6 @@
   name: Development Framework
   name_cn: 开发框架
   items:
-    - android-development-framework # 安卓开发框架
```
Comment (eyehwan) [inline collections.yml:60]: When you delete android-development-framework, how to handle the projects existed under this item?

Comment (eyehwan) [reply collections.yml:60]: Two choice:
1. Leave it unchanged until you add android-component-and-projects.yml into collection.yml. 
2. Delete it and adding  android-component-and-projects.yml into collection.yml at the same time.

**Review by eyehwan — CHANGES_REQUESTED** [review]

**Comment by hncslwx** [comment]
@eyehwan I agree with your choice 1. 
@QYFabc is working on tier 2 classes and normally there are a lot of tier 2 classes need be added before its completion, then I can add related tier 1 class.   so choice 1 is more flexible for cooprateion.

File: collections.yml (hunk 1 above)
Comment (hncslwx) [reply collections.yml:60]: @eyehwan I agree with your choice 1.
@QYFabc is working on tier 2 classes and normally there are a lot of tier 2 classes need be added before its completion, then I can add related tier 1 class. so choice 1 is more flexible for cooprateion.

**Pull request closed without merging**